sentry-ruby/spec/sentry/client/event_sending_spec.rb

```diff
@@ -246,6 +255,26 @@
 
         subject.send_event(event)
       end
+
+      it "applies before_send_transaction callback before sending the event" do
+        configuration.before_send_transaction = lambda do |event, _hint|
+          if event.is_a?(Sentry::Event)
```

Comment:
I think we can be more specific to use `Sentry::TransactionEvent` here and below?